sandbox/grommet-app/src/themes/theme.jsx

```diff
@@ -1032,8 +1032,8 @@ const buildTheme = (tokens, flags) => {
           offset: global.hpe.focusIndicator.outlineOffset,
         },
         shadow: {
-          color: focusBoxShadowParts[focusBoxShadowParts.length - 1],
-          size: focusBoxShadowParts[focusBoxShadowParts.length - 2],
+          color: global.hpe.focusIndicator.boxShadow.color, // focusBoxShadowParts[global.hpe.focusIndicator.boxShadowParts.length - 1],
+          size: global.hpe.focusIndicator.boxShadow.spread, // focusBoxShadowParts[focusBoxShadowParts.length - 2],
```

Comment:
Context for other reviewers https://github.com/grommet/grommet-theme-hpe/pull/549